apps/openmw/mwscript/miscextensions.cpp

```diff
@@ -700,6 +700,8 @@ namespace MWScript
 
                 virtual void execute (Interpreter::Runtime& runtime)
                 {
+                    MWWorld::Ptr ptr = R()(runtime);
+                    MWBase::Environment::get().getWorld()->queueMovement(ptr, osg::Vec3f(0, 0, -0.1f));
```

Comment:
Why the need to implement 'fall'? As far as I can tell falling actors should still be detected anyway because of the ray test in updateIdle().

Reply:
Just for sure.
IMHO, keeping placehoilders is not a good thing too.